### PR TITLE
[CHORE] Code source reorganization for improved readability.

### PR DESCRIPTION
**Pull request by Micsk10**
# Description

"I have made modifications primarily focused on improving the visual appearance of the source code. These changes do not have a major impact on the logic or functionality of the game, but they make the code cleaner, more readable, and aesthetically pleasing.

Fixes #issue

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

# Checklist:

- [ ] Title of the PR is relevant and is prefixed with [FEAT], [CHORE] or [FIX]
- [ ] Screenshot if it includes any UI changes
- [ ] I have read the contributing guidelines and agree to the T&Cs
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: src/features/game/types/collectibles.ts [hunk 1]
```diff
@@ -334,54 +337,56 @@ export const GOBLIN_BLACKSMITH_ITEMS: (
...
         "A whimsical, fungi-abode where the walls sprout with charm and even the furniture has a 'spore-tacular' flair!",
-      ingredients: {
-        "Wild Mushroom": new Decimal(50),
-        Gold: new Decimal(10),
-      },
       // 50 Team supply + giveaways
       supply: 2000 + 50,
       sfl: SFLDiscount(state, new Decimal(50)),
       boost: "+0.2 Wild Mushroom",
+      // Placeholders
```
Comment (brian-man) [inline src/features/game/types/collectibles.ts:344]: This was added (not sure why).  Should remove.

File: src/features/game/types/collectibles.ts [hunk 2]
```diff
@@ -334,54 +337,56 @@ export const GOBLIN_BLACKSMITH_ITEMS: (
...
       // only available when SEASONS["DAWN_BREAKER"] starts
       disabled: getCurrentSeason() !== "Dawn Breaker",
     },
     Maximus: {
       description: "Squash the competition with plump Maximus",
+      // 50 Team Supply + giveaways
+      supply: 350 + 50,
+      sfl: SFLDiscount(state, marketRate(20000)),
+      boost: "+1 Eggplant",
       // Placeholders
```
Comment (brian-man) [inline src/features/game/types/collectibles.ts:358]: Stale, remove

File: src/features/game/types/collectibles.ts [hunk 3]
```diff
@@ -334,54 +337,56 @@ export const GOBLIN_BLACKSMITH_ITEMS: (
...
-      boost: "+1 Eggplant",
       disabled: getCurrentSeason() !== "Dawn Breaker",
     },
     Obie: {
       description: "A fierce eggplant soldier",
+      // 100 Team Supply + Giveaways
+      supply: 2500 + 100,
+      sfl: SFLDiscount(state, marketRate(2000)),
+      boost: "25% faster eggplants",
       // Placeholders
```
Comment (brian-man) [inline src/features/game/types/collectibles.ts:371]: Placeholders comment - Stale, remove.

File: src/features/game/types/collectibles.ts [hunk 4]
```diff
@@ -334,54 +337,56 @@ export const GOBLIN_BLACKSMITH_ITEMS: (
...
       disabled: getCurrentSeason() !== "Dawn Breaker",
     },
     "Purple Trail": {
       description:
         "Leave your opponents in a trail of envy with the mesmerizing and unique Purple Trail",
+      
+      sfl: SFLDiscount(state, marketRate(800)),
+      supply: 10000,
+      boost: "+0.2 Eggplant",
       // Placeholders
```
Comment (brian-man) [inline src/features/game/types/collectibles.ts:385]: Placeholders comment - Stale, remove.

**Comment by brian-man** [comment]
The `translate` changes seem unrelated to this PR and should really be separated into a different PR.

**Comment by elias7905** [comment]
> The `translate` changes seem unrelated to this PR and should really be separated into a different PR.

@brian-man the translate probably came from mmytranslations library branch somehow, will help him correct them for him

**Review by elias7905 — APPROVED** [review]
Alright fixed the errors that were present in this PR

**Review by adamhannigan — APPROVED** [review]
Looks good to me! (LGTM)

Thanks for the contribution 🤗

**Pull request merged**